k8s/namespaces/em/em-hrs-api/aat.yaml

```diff
@@ -2,8 +2,11 @@ apiVersion: helm.fluxcd.io/v1
 kind: HelmRelease
 metadata:
   name: em-hrs-api
+  annotations:
+    fluxcd.io/tag.java: glob:pr-375-*
```

Comment:
Needs few more lines here like this - https://github.com/hmcts/cnp-flux-config/blob/15f738efe5726d29dd8df3731be05c0eb353c2af/k8s/namespaces/em/em-anno/demo.yaml#L8

Reply:
it got rejected earlier with that line...

Reply:
https://github.com/hmcts/cnp-flux-config/runs/3621962647?check_suite_focus=true

docmosis
---
em-hrs-api'
+ HELMRELEASE_CHECK=em-hrs-api
+ '[' em-hrs-api == null ']'
+ '[' em-hrs-api == '' ']'
+ '[' 0 -eq 0 ']'
+ echo 'Non whitelisted HelmReleases found with hmcts.github.com/prod-automated annotation in k8s/aat/common-overlay'
+ exit 1
Error: Process completed with exit code 1.